assets/src/edit-story/app/currentUser/currentUserProvider.js

```diff
@@ -43,11 +43,21 @@ function CurrentUserProvider({ children }) {
     [_updateCurrentUser]
   );
 
+  const toggleWebStoriesMediaOptimization = useCallback(() => {
+    return updateCurrentUser({
+      meta: {
+        web_stories_media_optimization: !currentUser.meta
+          .web_stories_media_optimization,
+      },
+    });
+  }, [currentUser, updateCurrentUser]);
```

Comment:
Might want to use this here as well - https://github.com/google/web-stories-wp/blob/da23d65cbca76c604350464f0538115d280a7a06/assets/src/edit-story/components/library/panes/media/local/videoOptimizationDialog.js#L53-L60

Reply:
@spacedmonkey I may be wrong, but it seems like we don't need this since we're not using that modal, right? If it's just the toggle, then don't we only need one action to toggle the user's value?

Reply:
@samwhale I think what he's saying is that this new `toggleWebStoriesMediaOptimization` callback can be use in this dialog component. To make things DRY.